samba: correct log files location

We use /var/log/samba.

Upstream-Status: Pending

Signed-off-by: Joe Slater <joe.slater@windriver.com>

--- a/packaging/RHEL/setup/smb.conf
+++ b/packaging/RHEL/setup/smb.conf
@@ -63,7 +63,7 @@
 
 # this tells Samba to use a separate log file for each machine
 # that connects
-   log file = /usr/local/samba/var/log.%m
+   log file = /var/log/samba/log.%m
 
 # Put a capping on the size of the log files (in Kb).
    max log size = 50
